src/main/java/org/jitsi/videobridge/cc/vp8/VP8Frame.java

```diff
@@ -93,6 +93,13 @@
      */
     private boolean endingSequenceNumberIsKnown = false;
 
+    /**
+     * The max sequence number that was seen before the arrival of the first
+     * packet of this frame. This is useful for piggybacking any mis-ordered
+     * packets.
+     */
+    private final int maxSequenceNumberSeenBeforeFirstPacket;
```

Comment:
so this is `the highest sequence number we've seen from a packet from this stream before seeing a packet that belonged to this frame`?

Reply:
No, this is the highest sequence number of _this_ frame that was seen before the arrival of its first packet. For example, if a frame comprised of packets 1,2,3 is received as 2,1,3 then this field would be equal to 2.

Reply:
when we need to figure out how many to piggy-back, can we not just start at the first sequence number of the frame and increment up, checking each time if `vp8frame.matches(packet sequence number)`? and avoid this extra state?  this way may be simpler, i dunno, just a thought

Reply:
That's almost how it worked in one of your previous reviews but for some reason some queue in the bridge started to fill up and drop packets..